tests/e2e.spec.ts

```diff
@@ -0,0 +1,149 @@
+import { test, expect, Page } from '@playwright/test';
+
+const QUIZ_TITLE = 'Solvro Rulez';
+
+async function loginAsGuest(page: Page) {
```

Comment:
trochę szkoda, że nie użyłeś POMa (Page Object Model) by zamodelować te rzeczy, dobra praktyka a ty masz to w zasadzie gotowe